app/src/main/java/com/example/mytrainingpal/components/YoutubeDialog.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.mytrainingpal.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+
+
+//Adapted from https://foso.github.io/Jetpack-Compose-Playground/material/alertdialog/
+
+@Composable
+fun YoutubeDialog(openDialog: Boolean, dismissClick: () -> Unit) {
+    MaterialTheme {
+        Column {
+            val openDialog = remember { mutableStateOf(openDialog) }
+            if (openDialog.value) {
+                AlertDialog(
+                    onDismissRequest = {
+                        // Dismiss the dialog when the user clicks outside the dialog or on the back
+                        // button. If you want to disable that functionality, simply use an empty
+                        // onCloseRequest.
+                        openDialog.value = false
+                    },
+                    title = {
+                        Text(text = "Dialog Title")
+                    },
+
+                    confirmButton = {
+                        Button(
+
+                            onClick = {
+                                openDialog.value = false
+                            }) {
+                            Text("This is the Confirm Button")
```

Comment:
I would rename the val openDialog, since it is already the name of the parameter
```suggestion
            val showDialog = remember { mutableStateOf(openDialog) }
            if (showDialog.value) {
                AlertDialog(
                    onDismissRequest = {
                        // Dismiss the dialog when the user clicks outside the dialog or on the back
                        // button. If you want to disable that functionality, simply use an empty
                        // onCloseRequest.
                        showDialog.value = false
                    },
                    title = {
                        Text(text = "Dialog Title")
                    },

                    confirmButton = {
                        Button(

                            onClick = {
                                showDialog.value = false
                            }) {
                            Text("This is the Confirm Button")
```

Reply:
oh this should be gone 🥲